### PR TITLE
12.0 pos order print check

### PR DESCRIPTION
**Pull request by trojikman**

### CODE REVIEW

File: pos_order_print_check/views/assets_demo.xml [hunk 1]
```diff
@@ -1,15 +0,0 @@
...
-<odoo>
-    <template
-        id="demo_assets_backend"
-        name="pos_order_print_check demo assets"
-        inherit_id="web.assets_backend"
-    >
-        <xpath expr="." position="inside">
-            <script
-                type="text/javascript"
-                src="/pos_order_print_check/static/src/js/tour_pos_order_print_check.js"
```
Comment (yelizariev) [inline pos_order_print_check/views/assets_demo.xml:11]: Tests give negative result, because tour is not attached

Comment (trojikman) [reply pos_order_print_check/views/assets_demo.xml:11]: Nothing has changed after turning on tour

Comment (trojikman) [reply pos_order_print_check/views/assets_demo.xml:11]: Added TODO in Roadmap

**Comment by trojikman** [comment]
I approve to merge it now

File: pos_order_print_check/views/assets_demo.xml (hunk 1 above)
Comment (yelizariev) [reply pos_order_print_check/views/assets_demo.xml:11]: Ты не в тот асетс засунул тур

https://github.com/trojikman/pos-addons/blob/0428ec636ed5da36d02c3da44ac6489bfd788269/pos_order_print_check/views/pos_order_print_check_template.xml#L8

screenshot-github com-2020 05 15-10_18_20

**Pull request merged**